apps/next15-demo/src/app/[[...slug]]/page.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { SWRProvider } from "../components/SWRProvider";
+import { BridgeHandler } from "../components/BridgeHandler";
+import { client } from "@/lib/client";
+import { Story } from "../Story";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import { getStory } from "@/lib/getStory";
+
+export default async function Page({ params }: { params: { slug: string } }) {
+  let { slug } = await params;
+
+  if (slug == null) {
+    slug = "home";
+  }
+
+  if (Array.isArray(slug)) {
+    slug = slug.join("/");
+  }
+
+  const { story, header } = await fetchPage(slug);
+
+  return (
+    <SWRProvider
+      fallback={{
+        [`cdn/stories/${slug}`]: story,
+        "cdn/stories/site-config": header,
```

Comment:
The cache keys simply need to match the request params for useSWR. Similar patterns exist in other data fetching libraries